### PR TITLE
Proper code highliting and intendation everywhere for README.md

### PR DESCRIPTION
**Pull request by Anton-Latukha**
This is the maximum what can be done today on the closed-source GitHub engine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -290,29 +302,40 @@ For example, to include line numbers for a `assertEquals()` function call, repla
...
+
+# load shunit2
+. ../src/shell/shunit2
+```
 
 Notes:
 
-1. Due to how shell parses command-line arguments, all strings used with macros should be quoted twice. Namely, single-quotes must be converted to single-double-quotes, and vice-versa. If the string being passed is absolutely for sure not empty, the extra quoting is not necessary.<br /><br />Normal `assertEquals` call.<br/><br/>`assertEquals 'some message' 'x' ''`<br/><br/>Macro `_ASSERT_EQUALS_` call. Note the extra quoting around the _message_ and the _null_ value.
+1. Due to how shell parses command-line arguments, all strings used with macros should be quoted twice. Namely, single-quotes must be converted to single-double-quotes, and vice-versa. If the string being passed is absolutely for sure not empty, the extra quoting is not necessary.
+   Normal `assertEquals` call.
```
Comment (cbandy) [inline README.md]: The original rendering had an empty line above this one. Is it possible to recreate it here?

**Comment by Anton-Latukha** [comment]
Here, all review points made.

**Comment by kward** [comment]
Hi, I found this issue only after having mades some major revisions to the formatting of the doc. If you would like to review the changes and resubmit a pull for any fixes, please do so.

For one, I noticed that you provided the Linguist hint of 'zsh' for some of the zsh shell. I didn't do that (I wasn't aware of zsh support in Linguist), and would take patches for that and other things you find.

**Comment by Anton-Latukha** [comment]
`zsh` marks - are `Shell` highlighting and also future proofing.

What Linguist supports can be seen at:
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

There we can see that `zsh` is just an alias for `Shell`. But in future if split and support of `zsh` as other languages emerge, of course it would be marked `zsh`, so why not use it if that is right marking.

Maybe I will do rebase and new push, if I get to it.

**Comment by Anton-Latukha** [comment]
Ok.

This is cosmetic.

And now upstream is changed.

**Pull request closed without merging**